test/module-path-tests/pom.xml

```diff
@@ -0,0 +1,168 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2010-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License").
+  ~ You may not use this file except in compliance with the License.
+  ~ A copy of the License is located at
+  ~
+  ~  http://aws.amazon.com/apache2.0
+  ~
+  ~ or in the "license" file accompanying this file. This file is distributed
+  ~ on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+  ~ express or implied. See the License for the specific language governing
+  ~ permissions and limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>aws-sdk-java-pom</artifactId>
+        <groupId>software.amazon.awssdk</groupId>
+        <version>2.0.0-preview-13-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>module-path-tests</artifactId>
+    <!-- Setting it to a different version so it can always load dependencies from maven
+    rather than local module-->
+    <version>1.0.0-SNAPSHOT</version>
```

Comment:
Why do we want to always load from maven?

Reply:
Because Intellij will always try to load dependencies from local module instead of maven and it shows a bunch of errors saying "module not found" because those modules don't have `Automatic-Module-Name` at compile time.

![screen shot 2018-10-26 at 12 51 11 pm](https://user-images.githubusercontent.com/33073555/47589383-d70c8100-d91d-11e8-9c74-2120e503f048.png)